qubesmanager/table_widgets.py

```diff
@@ -281,26 +264,33 @@ def update_vm_state(self):
         self.upd_info.update_outdated()
 
 
-class VmTemplateItem(QtGui.QTableWidgetItem):
-    def __init__(self, vm):
-        super(VmTemplateItem, self).__init__()
+class VMPropertyItem(QtGui.QTableWidgetItem):
+    def __init__(self, vm, property_name, empty_function=(lambda x: False),
+                 check_default=False):
+        super(VMPropertyItem, self).__init__()
         self.setFlags(QtCore.Qt.ItemIsSelectable | QtCore.Qt.ItemIsEnabled)
+        self.setTextAlignment(QtCore.Qt.AlignVCenter)
         self.vm = vm
         self.qid = vm.qid
+        self.property_name = property_name
         self.name = vm.name
-        self.setTextAlignment(QtCore.Qt.AlignVCenter)
+        self.empty_function = empty_function
+        self.check_default = check_default
         self.update()
 
     def update(self):
-        if getattr(self.vm, 'template', None) is not None:
-            self.setText(self.vm.template.name)
+        val = getattr(self.vm, self.property_name, None)
+        if self.empty_function(val):
+            self.setText("")
+        elif val is None:
+            self.setText("n/a")
+        elif self.check_default and \
+                self.vm.property_is_default(self.property_name):
+            self.setText('default (' + str(val) + ')')
```

Comment:
This, on bool property will be inconsistent with check_default=False. One with show `default (True)` and the other `Yes`.